src/SprykerEco/Yves/Adyen/Form/CreditCardSubForm.php

```diff
@@ -35,6 +33,11 @@ class CreditCardSubForm extends AbstractSubForm
 
     protected const PAYMENT_METHOD = 'credit-card';
 
+    protected const NOT_VALID_CARD_NUMBER_MESSAGE = 'Card number is not valid.';
+    protected const NOT_VALID_EXPIRY_YEAR_MESSAGE = 'Card expiry year is not valid.';
+    protected const NOT_VALID_EXPIRY_MONTH_MESSAGE = 'Card expiry month is not valid.';
+    protected const NOT_VALID_SECURITY_CODE_MESSAGE = 'Security coder is not valid.';
```

Comment:
These error messages should be glossary keys to enable translations for the projects.

Reply:
Also, I do not quite agree with what the messages are saying: technically you are validating for emptiness and saying the data is not valid in a message. And I do get the empty will mean the js did not fill the fields because of the validation.

Are there any requirements for these messages from the product side? Should we be able to give different messages for emptiness and data validity?

Reply:
Please prefix the consts with `GLOSSARY_KEY_CONSTRAINT_MESSAGE_`

Reply:
```
    protected const NOT_VALID_CARD_NUMBER_MESSAGE = 'adyen.payment.error.cc_number';
    protected const NOT_VALID_EXPIRY_YEAR_MESSAGE = 'adyen.payment.error.cc_year';
    protected const NOT_VALID_EXPIRY_MONTH_MESSAGE = 'adyen.payment.error.cc_month.';
    protected const NOT_VALID_SECURITY_CODE_MESSAGE = 'adyen.payment.error.cc_cvv';
```

Reply:
> Please prefix the consts with `GLOSSARY_KEY_CONSTRAINT_MESSAGE_`.

This is not resolved.